src/scss/custom.scss

```diff
@@ -510,7 +533,6 @@ ol {
 
   &__links {
     padding-bottom: 2.6rem;
-    max-width: 122rem;
```

Comment:
Removing the footer links `max-width` makes the footer context expand across the entire screen width. This might be OK for average screen widths, but for extra wide screens I think it might be very unreadable.

The following screenshots are from my 1920x1200 monitor.

Without `max-width`:
![Screenshot 2025-03-05 at 7 55 58 AM](https://github.com/user-attachments/assets/4c7ff271-1c0c-4fe5-a132-33a1ead46201)

With `max-width`:
![Screenshot 2025-03-05 at 7 55 43 AM](https://github.com/user-attachments/assets/daa705a6-09e1-46e8-937c-bbf5ef8e399b)